rules/cross-platform/data_destruction_meow_attack_detected.toml

```
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2020/08/10"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/10"
+
+[rule]
+author = ["Elastic"]
+description = """
+The Meow attack targets insecurely configured Elasticsearch, MongoDB, and Cassandra databases. The attack destroys the
+contents of the database and replaces (or appends) the word "meow" to the database name.
+"""
+false_positives = [
+    """
+    The word "meow" used in database names will trigger this detection rule. Incident Responders should validate the
+    database is expected and modify this rule as appropriate.
+    """,
+]
+index = ["filebeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Meow Database Attack Detected"
```

detected is already implied with rules, so I think we can remove that word. maybe this?
```suggestion
name = "Potential Meow Database Attack"
```